.dockerignore

```diff
@@ -0,0 +1,16 @@
+.cover
+.dockerignore
```

Comment:
Removed `.git`, as it's used by the `server.Dockerfile` to generate the `version` output. We could consider doing that outside of the container, but that may complicate building the Dockerfile (e.g. through docker hub autobuild)